### PR TITLE
feat(subgraph) - set key manager when key is mined

### PR DESCRIPTION
**Pull request by kalidiagne**
# Description

# Issues

Fixes #
Refs #

# Checklist:

- [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: subgraph/src/helpers.ts [hunk 1]
```diff
@@ -75,3 +75,16 @@ export function loadOrCreateUnlockDailyData(
 
   return unlockDailyData
 }
+
+export function getKeyManagerOf(
+  lockAddress: Address,
+  tokenId: BigInt,
+  owner: Address
+): Address {
+  const lockContract = PublicLockV7.bind(lockAddress)
```
Comment (julien51) [inline subgraph/src/helpers.ts:84]: I am a bit confused about why you use `PublicLockV7` here? But i think that could work? @clemsos wdyt?

**Review by julien51 — APPROVED** [review]
I _think_ that is good. Would love to get a quick check by @clemsos

File: subgraph/src/helpers.ts (hunk 1 above)
Comment (kalidiagne) [reply subgraph/src/helpers.ts:84]: there is no difference in the implementation of `keyManagerOf` from `v7` and `v11`, will works fine

**Review by clemsos — APPROVED** [review]
LGTM

**Pull request merged**